crates/rooch-types/src/transaction/ethereum.rs

```diff
@@ -161,8 +125,7 @@ impl AbstractTransaction for EthereumTransaction {
             //TODO should change the seqence_number to u256?
             seqence_number: self.0.nonce.as_u64(),
             authenticator: Authenticator::ecdsa_recoverable(
-                // TODO need to support handling ethereum signature to Rooch signature
-                self.convert_eth_signature_to_recoverable_secp256k1_signature()
+                self.convert_eth_transaction_signature_to_rooch_signature()
```

Comment:
We should not convert the signature in Rust. We need to put the original ethereum signature and decode it in the Move contract.

Reply:
> We should not convert the signature in Rust. We need to put the original ethereum signature and decode it in the Move contract.

The Ethereum transaction contains original signature R, S, V, and I did put them into constructing Rooch signature so they're the same signature bytes in Rust and can be decoded in Move.

Reply:
Or I should rename it to `self.into_signature` for better understanding?

Reply:
Originally,

`Ethereum Transaction` -> `Ethereum Signature`.

Now,

`Ethereum Transaction` -> `Rooch Ethereum Signature`.

Reply:
As the dev meeting discussion, the rooch native validator & signature is ed25519. `Rooch Ethereum Signature` is meanless.